README.md

```diff
@@ -16,7 +16,14 @@
 </div>
 <br>
 
-<img src="https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.png" height="75" alt="vim-plug">[![build](https://img.shields.io/github/actions/workflow/status/junegunn/vim-plug/test.yml?branch=master)](https://github.com/junegunn/vim-plug/actions/workflows/test.yml?query=branch%3Amaster)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./plug-dark.png">
+  <img src="./plug.png" height="75" alt="vim-plug">
+</picture>
+<a href="https://github.com/junegunn/vim-plug/actions/workflows/test.yml?query=branch%3Amaster">
+  <img src="https://img.shields.io/github/actions/workflow/status/junegunn/vim-plug/test.yml?branch=master">
+</a>
+
```

Comment:
```suggestion
```

Reply:
Hmm, it's still there.

Reply:
The triple sign indicates that the line above is an h1 header. If we just remove it, we'll lose the underline below the image.

<img width="867" alt="image" src="https://github.com/junegunn/vim-plug/assets/700826/841ed1a6-22be-4d58-9052-bb2d0b1800e6">


So I replaced it with an actual h1 tag.

Reply:
Understood, sorry about that.